src/plugins/home/public/application/components/guided_onboarding/getting_started.test.tsx

```diff
@@ -119,7 +122,7 @@ describe('getting started', () => {
     });
 
     const guideFilters = component!.find('[data-test-subj="onboarding--guideFilters"]');
-    expect((guideFilters.props() as GuideFiltersProps).activeFilter).toBe('all');
+    expect((guideFilters.props() as GuideFiltersProps).activeFilter).toBe('search');
```

Comment:
Is this test for the new flow now? Should we add another test for "classic" (I'd expect it to stay "all" in that case).

Reply:
This is smart but Im not sure how to approach this - I think in a later iteration and future PR I will be using the testing-react-library which might make this easier to test.

Reply:
I'm OK to leave as-is for now, as the classic flow is technically not being used until A/B testing is implemented. That said, once we introduce A/B testing, I would want to revisit this as we essentially lost the test coverage here for the classic flow.